packages/babel-plugin-transform-runtime/test/fixtures/absoluteRuntime/true-corejs3-proposals/output.js

```diff
@@ -1,6 +1,6 @@
 var _regeneratorRuntime = require("<CWD>/packages/babel-runtime-corejs3/regenerator");
 
-var _mapInstanceProperty = require("<CWD>/packages/babel-runtime-corejs3/core-js/instance/map");
+var _mapInstanceProperty = require("<CWD>/packages/babel-runtime-corejs3/core-js/instance/map.js");
```

Comment:
Here the extension is correct, because when using absolute or relative paths npm ignores `package.json#exports`